test/contracts/MarginBase.t.sol

```diff
@@ -48,11 +225,376 @@ contract MarginAccountFactoryTest is DSTest {
     }
 
     /// @dev Deposit/Withdrawal fuzz test
-    function testWithdrawal(uint amount) public {
+    function testWithdrawal(uint256 amount) public {
+        cheats.assume(amount > 0);
```

Comment:
amount should always be above 0 because it's a UINT, were you testing INTs?

Reply:
I should've commented on this but the amount can't be "0". MarginBase emits deposit/withdraw events indexing user addresses and we don't want them to potentially be spammed.